### ISSUE TITLE
Toolbar

### ISSUE THREAD

**Issue by Pinuaple** [mydockfinder/mydockfinder#692]
the toolbar in figma krita and inkscape doesnt work
Inkscape directly doesnt show up the buttons

**Comment by mydockfinder** [comment]
Yes, because when I was asked to adapt to figma, it just asked me to add text and didn't let me add features




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年5月11日(星期六) 凌晨0:36
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; 
主题: [mydockfinder/mydockfinder-for-Win10-Win11] Figma toolbar (Issue #692)














 
the toolbar in figma doesnt work
